### ISSUE TITLE
picturechooser always show in portrait orientation

### ISSUE THREAD

**Issue by TingtingAn** [MvvmCross/MvvmCross#761]
In my ipad project , i use the picturechooser to select an image. my app ui orientation is landscape,but picturechooser always show in portrait orientation can the picturechooser open by the landscape orientation way?

**Comment by PenaPP** [comment]
I have a landscape iOS application for iPad and when I try to use the ChoosePictureFromLibrary method it always crashes with the following exception...
```
Objective-C exception thrown.
Name: UIApplicationInvalidInterfaceOrientation
Reason: Supported orientations has no common orientation with the application, and [PUUIAlbumListViewController shouldAutorotate] is returning YES
```

I guess it is related to the jamesatgithub's comment about popovers and modality :(

**Comment by PenaPP** [comment]
Confirmed... it has to be in popover to work in landscape mode ;)

**Comment by martijn00** [comment]
@PenaPP Can you make a PR to fix this?

**Comment by PenaPP** [comment]
@martijn00 I guess I don't have any :) It wasn't bug in the library, but in my code ;)

**Comment by PenaPP** [comment]
@martijn00 I guess I don't have any. I solved it by changing my code ;)

**Comment by Jerome2606** [comment]
Hello, 
Picture chooser is for me one very important feature when we speak about mobile.
I'm very happy with mvvmcross to separate the logic of the ui and make my application unit testing ready and share a lot of code.
But I'm very disappointing about this bug that blocks me for 3 days before getting on this thread that we should be in portrait mode and not landscape or be in a "popover". I doesn't understand what does it mean.

Is anybody is able to take time to explain me what code is needed to fix this issue or fix the bug inside of the plugin plz ???

Thank you

**Comment by Cheesebaron** [comment]
@Jerome2606 Is this on iPad or what? Popovers are only necessary on iPad, and we currently don't have a way of passing along one of those to present the `UIImagePickerController`.

One thing that puzzles me is how you can be stuck with this for 3 days, when you always had the possibility of showing your own `UIImagePickerController` ¯\_(ツ)_/¯

**Comment by Jerome2606** [comment]
Yes it is for ipad.

What I mean for 3 days is that I'm starting with Xamarin and iOS development and was not able to know why suddenly my app was crashing. It took me a while to find these ways to get crash info:
https://www.utest.com/articles/get-ios-logs-on-windows-using-the-command-line
https://support.hockeyapp.net/kb/client-integration-ios-mac-os-x-tvos/how-to-manually-upload-and-symbolicate-a-crash

After that I get the error and this guides me to this thread.
it is again difficult for me to understand all the things into iOS Xamarin and MvvmCross framework.

So basically the solution we start to implement is like you said implement our own UIImagePickerController but at this point I'm not able to understand how to implement "popover" to make it working in landscape so for now we force user to use portrait and that should be only a "workaround". 

I would like to get some existing code like @PenaPP did on his side.

**Comment by Cheesebaron** [comment]
See: https://forums.xamarin.com/discussion/comment/16659#Comment_16659

**Comment by Jerome2606** [comment]
Thank you, I take a look on that.
